SAKPaaS/src/app/app.module.ts

```diff
@@ -54,7 +56,7 @@ import { LocationDetailsModule } from 'src/app/components/location-details/locat
     OccupancyViewModule,
     SearchBarModule,
     SharedModule,
-    SnackBarModule
+    SnackBarModule,
```

Comment:
Why is there a comma 🤔

Reply:
Trailing commas sind doch normal? Werden ja eh ignoriert 😂